test/cli/suggest-sig-override-edge/suggest-sig-override-edge.rb

```diff
@@ -69,7 +72,7 @@ def nope; end
 
 # Doesn't need generated because parent already has overridable
 class DoesntNeedGenerated
-  sig {implementation.overridable.void}
+  sig {override.overridable.void}
   def already_overridable; end
 end
```

Comment:
Not at all necessary for this PR, but we should probably change this test at some point now that `generated` is a thing of the past.